zsh/functions/whats-in-port

```diff
@@ -0,0 +1,23 @@
+# List process running on provided port
+#
+# whats_in_port 3000
+#
+# output:
+# COMMAND   PID   USER   FD   TYPE             DEVICE SIZE/OFF NODE NAME
+# ruby    25583   root   11u  IPv4 0xee20607697a79bf7      0t0  TCP *:irdmi (LISTEN)
+
+whats_in_port() {
```

Comment:
THANK ~GOD~ ALEJANDRO!

Reply:
I have a similar [function in my dotfiles](https://github.com/stevehanson/dotfiles/blob/main/zsh/functions/ports). Mine has a default argument of 3000 instead of erroring, but I think I like that yours fails if no argument is provided. 3000 makes sense for me but maybe not for other users, so it'd be hard to find a perfect default.